### PR TITLE
Ignore/restructure

### PR DESCRIPTION
**Pull request by sventer444**
fixed the file structure issue I created, made an ignore file

### CODE REVIEW

**Comment by sventer444** [comment]
@GoodGuyGregory the git ignore missed something and I got an email about an api key being exposed, I assume it is in the firebase config file on the "baseline" branch pull request. I think you said something about knowing how to fix that issue so I requested you to fix the .gitignore on this branch

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+# Edit at https://www.gitignore.io/?templates=vue,node,vuejs,firebase
+
+### Firebase ###
+.idea
+**/node_modules/*
+**/.firebaserc
+
+### Firebase Patch ###
+.runtimeconfig.json
+.firebase/
```
Comment (GoodGuyGregory) [inline .gitignore:12]: This should Hide those files from the Github Watcher

**Review by GoodGuyGregory — APPROVED** [review]
I will need to look at your next PR to determine how it is giving you that error. To me the `gitignore` looks fine

**Comment by GoodGuyGregory** [comment]
Will merge we can always fix this later

**Pull request merged**